x-pack/plugins/observability/public/pages/rule_details/components/actions.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { EuiText, EuiSpacer, EuiFlexGroup, EuiFlexItem, EuiIcon, IconType } from '@elastic/eui';
+import { ActionsProps } from '../types';
+
+interface MapActionTypeIcon {
+  [key: string]: string | IconType;
+}
+const mapActionTypeIcon: MapActionTypeIcon = {
+  /* TODO:  Add the rest of the application logs (SVGs ones) */
```

Comment:
@fkanout I checked https://elastic.github.io/eui/#/display/icons and I tried to find rest logos, but I couldn't.  Instead of adding rest application logs, I was thinking maybe we have a more dynamic way to retrieve the logos from existing functionality. 

I will check out https://github.com/elastic/kibana/tree/main/x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types and see if we can get something from there.

Reply:
@fkanout I just investigated how we can get the icon dynamically and it looks like we can use the `actionTypeRegistry` from triggers_actions_ui to retrieve the iconClass. I quickly tested it out in your branch and it works. actionRegistry has a list method and we can get the iconClass from there!
<img width="1314" alt="Screenshot 2022-05-11 at 14 43 08" src="https://user-images.githubusercontent.com/2852703/167852491-4ffdfbef-7b6c-47f4-bc32-cfedb44cf69d.png">



```suggestion
    function getActionIconClass(actionGroupId?: string): string | undefined {
    const actionGroup = actionTypeRegistry.list().find((group) => group.id === actionGroupId);
    return actionGroup?.iconClass;
  }

```

`actionRegistry` we can get like this:

```
 const {
    triggersActionsUi: { ruleTypeRegistry, actionTypeRegistry },
  } = useKibana().services;
```


This is not a complete code (no Typescript), just a proof of concept that we can dynamically get what we need. Do you want me to wrap it up and push directly to your branch? Or shall I fix it on another PR once this PR is merged? Whatever works for you.

Reply:
Just for tracking purposes here's the issue that will fix this https://github.com/elastic/kibana/issues/132220